src/globalStyles.tsx

```diff
@@ -38,7 +38,7 @@ const globalStyles = (
         --text-4: clamp(2.07rem, calc(1.84rem + 1.17vw), 2.75rem);
         --text-5: clamp(2.49rem, calc(2.16rem + 1.64vw), 3.43rem);
 
-        --toilet-max-width: 845px;
+        --toilet-max-width: 1680px;
```

Comment:
Why did we change this value?

Reply:
It is a mildly arbitrary number I stole from the designers I work with. But because I wanted the `Center` component to be wider by default and have a max-width of `75ch`* for blogs and other text based content I edited this. I toyed with the idea of adding a global variable, something like `--toilet-width: 75ch`, maybe I should for clarity? 🤔 It rather hidden in that `Center` layout.

*This number does have rationality to it, taken from this article - https://piccalil.li/blog/line-length/

Reply:
I like it, but maybe use `ch` units directly and add a CSS comment as to why with a link to that reference?